public/locales/zh/translation.json

```diff
@@ -153,7 +153,7 @@
     "postalCode": "邮政编码",
     "sortingCode": "排序代码",
     "state": "州/省",
-    "userRegistrationRequired": "需要用户注册",
+    "Is Public": "是公开的",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.